SyntheticsRunTestsTask/utils.ts

```diff
@@ -18,3 +18,23 @@ export const getDefinedInteger = (value: string | undefined, {inputName}: {input
 
   return number
 }
+
+// Cannot use task.getBoolInput() because "If required is false and the value is not set, returns false."
+// We need to fallback to our defaults.
```

Comment:
```suggestion
// This helper needs to return undefined if the input is not provided so that it's overwritten by the other parameter sources when building the config.
```